hbase-client/src/main/java/org/apache/hadoop/hbase/ClusterStatus.java

```diff
@@ -165,6 +165,13 @@ public int getDeadServers() {
     return deadServers != null ? deadServers.size() : 0;
   }
 
+  /**
+   * @return map of the names of region servers on the live list with associated ServerLoad
+   */
+  public Map<ServerName, ServerLoad> getLiveServersLoad() {
```

Comment:
Additive change to Public interface is fine. We are already also changing RegionLoad.